.github/workflows/commit.yml

```diff
@@ -99,7 +103,7 @@ jobs:
       - name: deploy DHL
         uses: cloud-gov/cg-cli-tools@main
         with:
-          command: cf push --vars-file vars.development.yml --strategy rolling --no-wait
```

Comment:
Just a reminder that this is fine for dev (and maybe even staging), but we probably want to keep around for production...

Reply:
I removed this b/c CG build errors out when pushing multiple apps this way. I could fix this by making a multiline command though, which is what I'm going to do. Will take a bit to test.

Reply:
this is resolved